torch/export/decomp_utils.py

```diff
@@ -38,7 +45,8 @@ def __init__(self):
         self.decomp_table = _core_aten_decompositions_post_autograd()
 
         for op in _collect_all_valid_cia_ops_for_aten_namespace():
-            self.decomp_table[op] = _get_decomp_for_cia(op)
+            if op not in PRESERVED_ATEN_CIA_OPS:
```

Comment:
oh I think i see. Instead of making the change here, can we:

(1) use `torch._decomp.core_aten_decompositions()`

(2) tweak that function [here](https://github.com/pytorch/pytorch/blob/main/torch/_decomp/__init__.py#L298) - today, it looks like it unconditionally adds all CIA decomps to the list of decomps. But instead, we can filter it to not add the decomp to the list if the op has a "core" tag. Something like:
```
    for op in _collect_all_valid_cia_ops_for_aten_namespace():
        if torch.Tag.core not in op.tags:
            decomp_table[op] = _get_decomp_for_cia(op)
        
```

Reply:
Ah sorry the PR that makes core_aten_decomp as alias to export table is still not landed yet.... 
But i think we should do (2) yeah. Maintaining another list PRESERVED_ATEN_CIA_OPS makes the current decomp logic even more complicated.

Reply:
Ahh i see so you want to only do this for this op for this PR... Could you just hide this logic here https://www.internalfb.com/code/fbsource/[2f8dd11dd8963caf6e9f4ce48464ae111b55f1ad]/fbcode/caffe2/torch/_export/utils.py?lines=1072 Basically this is the only util function we use to query all valid cia ops we can mess qround. So we exclude it from this list, export system will automatically think this op is not a CIA i think...